intelmq/etc/feeds.yaml

```
@@ -1898,3 +1898,34 @@ providers:
             http_password: <your password>
         parser:
           module: intelmq.bots.parsers.fireeye.parser
+  Benkow:
+    Malware Panels Tracker:
+      description: Benkow Panels tracker is a list of fresh panel from various malware. The feed is available on the webpage: http://benkow.cc/passwords.php
```

```suggestion
      description: "Benkow Panels tracker is a list of fresh panel from various malware. The feed is available on the webpage: http://benkow.cc/passwords.php"
```
YAML doesn't like the colon